frontend/src/components/Tournament.js

```diff
@@ -12,18 +12,43 @@ import {
   Switch
 } from "solid-js";
 
-import { fetchTeams, fetchTournamentBySlug } from "../queries";
+import {
+  fetchTeams,
+  fetchTournamentBySlug,
+  fetchUserAccessByTournamentSlug
+} from "../queries";
+import { useStore } from "../store";
+import { fetchUserData } from "../utils";
 import Breadcrumbs from "./Breadcrumbs";
 
 const Tournament = () => {
   const params = useParams();
   const [teamsMap, setTeamsMap] = createSignal({});
+  const [_, { userFetchSuccess }] = useStore();
+  const [playingTeam, setPlayingTeam] = createSignal(null);
 
   const tournamentQuery = createQuery(
     () => ["tournament", params.slug],
     () => fetchTournamentBySlug(params.slug)
   );
   const teamsQuery = createQuery(() => ["teams"], fetchTeams);
+  const userAccessQuery = createQuery(
+    () => ["user-access", params.slug],
+    () => fetchUserAccessByTournamentSlug(params.slug)
+  );
+
+  onMount(() => {
+    setTimeout(() => initFlowbite(), 100);
+    setTimeout(() => initFlowbite(), 500);
+    setTimeout(() => initFlowbite(), 1000);
+    setTimeout(() => initFlowbite(), 3000);
+    setTimeout(() => initFlowbite(), 5000);
+    setTimeout(() => initFlowbite(), 8000);
+  });
+
+  onMount(() => {
```

Comment:
We are doing this to persist the login state when user arrives on Tournament Home page right? Or any other reason?

Reply:
So can we move this to Header.js so itll be appicable for tournament schedule, standings and team page as well?

Reply:
yeah, i'll move this to header file and push

Reply:
Done, can you check ?